azext_edge/edge/providers/check/base/pod.py

```diff
@@ -27,229 +33,197 @@ def decorate_pod_phase(phase: str) -> Tuple[str, str]:
 
 
 def evaluate_pod_health(
-    check_manager: CheckManager,
-    namespace: str,
-    target: str,
-    pod: str,
-    display_padding: int,
-    service_label: Optional[str] = None,
-    detail_level: int = ResourceOutputDetailLevel.summary.value,
-) -> None:
-    target_service_pod = f"pod/{pod}"
-    pods = get_namespaced_pods_by_prefix(prefix=pod, namespace=namespace, label_selector=service_label)
-
-    process_pod_status(
-        check_manager=check_manager,
-        target=target,
-        target_service_pod=target_service_pod,
-        pods=pods,
-        display_padding=display_padding,
-        namespace=namespace,
-        detail_level=detail_level,
-    )
-
-
-def process_pod_status(
     check_manager: CheckManager,
     target: str,
-    target_service_pod: str,
-    pods: List[V1Pod],
-    display_padding: int,
     namespace: str,
+    padding: int,
+    pods: List[V1Pod],
     detail_level: int = ResourceOutputDetailLevel.summary.value,
 ) -> None:
 
-    def _decorate_pod_condition(condition: bool) -> Tuple[str, str]:
-        if condition:
-            return f"[green]{condition}[/green]", CheckTaskStatus.success.value
-        return f"[red]{condition}[/red]", CheckTaskStatus.error.value
-
-    # TODO: Consolidate more using table view
     if not pods:
-        return add_display_and_eval(
-            check_manager=check_manager,
-            target_name=target,
-            display_text=f"{target_service_pod}* [yellow]not detected[/yellow].",
-            eval_status=CheckTaskStatus.warning.value,
-            eval_value=None,
-            resource_name=target_service_pod,
-            namespace=namespace,
-            padding=(0, 0, 0, display_padding),
-        )
-
-    for pod in pods:
-        target_service_pod = f"pod/{pod.metadata.name}"
+        return
```

Comment:
I can't remember if this was always the case with this PR - but did this method display a "no pods" message? Or is that up to the individual CR check now that it's removed?

Reply:
yeah now the no pod message will be under individual CR check, since we need the label to output the error message, meanwhile we don't want to pass the label down to evaluate_pod_health